asserts/pool.go

```diff
@@ -170,6 +200,53 @@ func (u *unresolvedRec) merge(at *AtRevision, gnum uint16, gr *internal.Grouping
 	}
 }
 
+type unresolvedSeqRec struct {
+	at       *AtSequence
+	grouping internal.Grouping
+
+	serializedLabel Grouping
+
+	err error
+}
+
+func (u *unresolvedSeqRec) groupingPtr() *internal.Grouping {
+	return &u.grouping
+}
+
+func (u *unresolvedSeqRec) label() Grouping {
+	return u.serializedLabel
+}
+
+func (u *unresolvedSeqRec) isAssertionNewer(a Assertion) bool {
+	return a.Revision() > u.at.Revision
+}
+
+func (u *unresolvedSeqRec) revision() int {
+	return u.at.Revision
+}
+
+func (u *unresolvedSeqRec) error() error {
+	return u.err
+}
+
+func (u *unresolvedSeqRec) exportTo(r map[Grouping][]*AtSequence, gr *internal.Groupings) {
+	serLabel := Grouping(gr.Serialize(&u.grouping))
+	// remember serialized label
+	u.serializedLabel = serLabel
+	r[serLabel] = append(r[serLabel], u.at)
+}
+
+func (u *unresolvedSeqRec) merge(at *AtSequence, gnum uint16, gr *internal.Groupings) {
+	gr.AddTo(&u.grouping, gnum)
+	// use highest revision & sequence
```

Comment:
we need to be more careful here,  we can't mix  Pinned and Unpinned probably. Also...

Reply:
Hmm good point but seems tricky to handle. AFAIU this isn't a realistic scenario, in which case it would fall into the TODO you suggested for addUnresolvedSeq and perhaps I should just prevent repeated calls about same sequence right away rather than leaving a TODO?..

Reply:
maybe the easiest thing for now is not to have a merge,  given that we will have much more precise scenarios, but to error in AddSequenceToUpdate and AddUnresolvedSeq if there's already an entry in unresolvedSequences ?

Reply:
Done.